components/event_dispatcher/introduction.rst

```diff
@@ -496,7 +496,7 @@ Eventi e ascolatori consapevoli del distributore
 ``EventDispatcher`` inietta sempre l'evento distribuito, il nome dell'evento
 e un riferimento a sé stesso agli ascoltatori. Questo può portare ad applicazioni
 avanzate per ``EventDispatcher``, incluse la possibilità per gli ascoltatori di distribuire altri eventi,
-il concatenamento degli eventi o anche il caricamento pigro di più ascoltatori
+il concatenamento degli eventi o anche il caricamento pigro ("lazy") di più ascoltatori
 nell'oggetto distributore. Ecco degli esempi:
```

Comment:
Tu hai sicuramente maggiore visibilità della documentazione per cui sta a
te decidere, in particolare sulla coerenza globale. La mia proposta
(intendo "lazy loading" e non solo "lazy") aveva a che fare con il fatto
che è la prima volta che leggo in italiano 'caricamento pigro' e mi suona
un po' ridicolo ma per chi già sa di cosa si tratta non c'e' problema.
Mentre il lettore italiano che non conosce cosa sia vedendo, la prima
volta, 'caricamento pigro  ("lazy loading")', comprenderebbe che si tratta
di un concetto specifico. Verrebbe aiutato per es. ad andarsi a leggere
http://en.wikipedia.org/wiki/Lazy_loading. Se si cerca su google
'caricamento pigro' non hai di fatto risultati proprio perche' questa
terminologia in italiano non viene al momento utilizzata.

Il giorno 22 marzo 2015 16:56, Massimiliano Arione <notifications@github.com
> ha scritto:

> In components/event_dispatcher/introduction.rst
> <https://github.com/garak/symfony-docs-it/pull/206#discussion_r26904325>:
>
> > @@ -496,7 +496,7 @@ Eventi e ascolatori consapevoli del distributore
> >  ``EventDispatcher`` inietta sempre l'evento distribuito, il nome dell'evento
> >  e un riferimento a sé stesso agli ascoltatori. Questo può portare ad applicazioni
> >  avanzate per ``EventDispatcher``, incluse la possibilità per gli ascoltatori di distribuire altri eventi,
> > -il concatenamento degli eventi o anche il caricamento pigro di più ascoltatori
> > +il concatenamento degli eventi o anche il caricamento pigro ("lazy") di più ascoltatori
> >  nell'oggetto distributore. Ecco degli esempi:
>
> Scusami, ma se dovessimo fare così per ogni termine tradotto, dovremmo
> ripetere il termine in inglese in ogni punto in cui viene citato la prima
> volta. Siccome sarebbe bello mantenere una certa coerenza, o lo facciamo
> ovunque o non lo facciamo da nessuna parte. E siccome probabilmente farlo
> ovunque non è pensabile, sarebbe bene non farlo da nessuna parte.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/garak/symfony-docs-it/pull/206/files#r26904325>.
>